### ISSUE TITLE
[Feature Request] Import Customer / Project names

### ISSUE THREAD

**Issue by amenk** [Razeeman/Android-SimpleTimeTracker#128]
Would a feature be in scope to import customer names or project names from a csv file or event an URL / API, eventually automatically and autocomplete the activate name from it?

The user case would be - in a small mechanical engineering business - to easily track time on projects and / or customers.

**Comment by berzernberg** [comment]
+1 on this one. This app is the best for time tracker in all it's features and UI, but I don't know how to properly track time on projects via this app.

**Comment by Razeeman** [comment]
Hello! Can you describe a bit more how would this import work? What should the csv file contain and what would happen after the import?

**Comment by amenk** [comment]
yes, for example from a CSV. After an import the cards could be clicked on to track the time
